atd-vze/src/views/Crashes/CrashDiagram.js

```diff
@@ -153,9 +154,9 @@ const CrashDiagram = ({ crashId, isCr3Stored, isTempRecord }) => {
       {diagramError &&
         (isTempRecord ? (
           <CardBody>
-            <p>
+            <Alert color="danger">
               Crash diagrams are not available for user-created crash records.
-            </p>
+            </Alert>
```

Comment:
would you set the color of this alert to `info`, and also add the `fa-info-circle` icon to left of the alert text? And remove the period (`.`) from this alert and all other alerts? 😇  sorry for the poor scoping on this